StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Controllers/PayWithLinkViewController-NewPaymentViewController.swift

```diff
@@ -78,7 +79,11 @@ extension PayWithLinkViewController {
         private func makeConfiguration() -> PaymentElementConfiguration {
             var configuration = context.configuration
             configuration.linkPaymentMethodsOnly = true
-            configuration.appearance = LinkUI.appearance
+
+            // reconfigure `appearance` to use `linkUIAppearance` so all non-Link-specific
+            // UI gets appropriately customized for our Link use cases.
+            configuration.appearance = configuration.linkUIAppearance
+
```

Comment:
I called this change out in slack as a bit of an awkward one here: https://lickability.slack.com/archives/C094P3BRBL1/p1753813784878079?thread_ts=1753798256.703019&cid=C094P3BRBL1